deployment/terraform/storage.tf

```diff
@@ -17,7 +17,16 @@ resource "aws_s3_bucket" "static" {
 
 resource "aws_s3_bucket" "storage" {
   bucket = "${lower("${var.environment}")}-pfb-storage-${var.aws_region}"
-  acl    = "private"
+  acl    = "public-read"
+  policy = "${data.aws_iam_policy_document.anonymous_read_storage_bucket_policy.json}"
+
+  cors_rule {
+    allowed_origins = ["*"]
+    allowed_methods = ["GET"]
```

Comment:
Might want to allow HEAD here as well. I'm not sure its strictly necessary, but it is classified as a read operation.